pkg/provider/gke/gke.go

```diff
@@ -44,8 +44,12 @@ import (
 
 // New is the GKE constructor.
 func New() *GKE {
+	DeploymentVars := make(map[string]string)
+	// Set PROVIDER_SERVICE Deployment variable to  LoadBalancer
```

Comment:
Does `PROVIDER_SERVICE` need to be something different here ? Also please end comments with a full stop :)

Reply:
DOne